### DISCUSSION TITLE
Pluvia Weather v1.6.1

### DISCUSSION THREAD

**Discussion by SpicyChair** [SpicyChair/pluvia_weather_flutter#58] [General]
- Fix display of text in search bar in location search screen (#57)
- Improve styling of location search screen

**Other Changes**
- Migrate deprecated widgets (FlatButton -> TextButton etc)

<hr /><em>This discussion was created from the release <a href='https://github.com/SpicyChair/pluvia_weather_flutter/releases/tag/'>Pluvia Weather v1.6.1</a>.</em>

**Comment by tejasraman** [comment]
The one thing which kept me from using this app instead of Geometric weather (other than widgets) was the broken search so.... but amazing app @SpicyChair
